styles/generic/_global.scss

```diff
@@ -28,4 +28,5 @@ body {
     overflow: hidden;
     clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
     clip: rect(1px, 1px, 1px, 1px);
-}
\ No newline at end of file
+    left: -999px
```

Comment:
Without this we get a weird thing which adds an extra scrollbar. see:
http://g.recordit.co/e4Nszf9M5J.gif